restapi.go

```diff
@@ -1382,7 +1382,10 @@ func (s *Session) ChannelMessageSendComplex(channelID string, data *MessageSend)
 
 		for i, file := range files {
 			h := make(textproto.MIMEHeader)
-			h.Set("Content-Disposition", fmt.Sprintf(`form-data; name="file%d"; filename="%s"`, i, quoteEscaper.Replace(file.Name)))
+			h.Set("Content-Disposition",
+				`form-data; name="file`+strconv.Itoa(i)+
+					`"; filename="`+quoteEscaper.Replace(file.Name)+
+					`"`)
```

Comment:
I agree with legolord on that point printf just take performances and on most cases can be replaced, properly however that's true that sometimes It could make the code a bit less readable.
In the pull request I think the only case where printf could have sense is this one.